otter_welcome_buddy/formatters/debug.py

```diff
@@ -0,0 +1,15 @@
+""" Formatters only for development """
+
+
+class Formatter:
+    """Dependency for debug messages"""
+
+    @staticmethod
+    def bot_is_ready():
+        """It indicates when the bot is up and running"""
+        return "Ready"
+
+    @staticmethod
+    def bot_is_ready_2():
+        """Place holder to avoid problems with pylint"""
```

Comment:
lol

Reply:
I will modify the rules to be able to avoid the necessity of documenting everything